rule-engine/rule-engine-components/src/test/java/org/thingsboard/rule/engine/metadata/TbGetTenantAttributeNodeTest.java

```diff
@@ -67,20 +67,17 @@ EntityId getEntityId() {
 
     @Test
     public void errorThrownIfCannotLoadAttributes() {
-        mockFindUser(user);
         errorThrownIfCannotLoadAttributes(user);
     }
 
     @Test
     public void errorThrownIfCannotLoadAttributesAsync() {
-        mockFindUser(user);
         errorThrownIfCannotLoadAttributesAsync(user);
     }
 
     @Test
```

Comment:
I think that this test should be renamed to failedChainUsedIfTenantIdFromCtxCannotBeFound.